crowbar_framework/app/assets/stylesheets/content/panel.scss

```diff
@@ -0,0 +1,56 @@
+.panel-heading {
+  border-bottom: 0;
+}
+
+.panel .panel-heading h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
+  border: 1px;
+  border-color: #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.